man/Makefile.am

```diff
@@ -0,0 +1,42 @@
+# examples/Makefile.am
+#
+# Copyright (C)      2015 Carsten Schoenert <c.schoenert@t-online.de>
```

Comment:
I'd suggest you use your name for the copyright holder as you the author of the new file.